modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -511,10 +517,14 @@ struct CvCapture_FFMPEG
 
     AVPacket          packet;
     Image_FFMPEG      frame;
-    struct SwsContext *img_convert_ctx;
-
+    SwsContext      * img_convert_ctx;
     int64_t frame_number, first_frame_number;
 
+#if LIBAVUTIL_BUILD >= CALC_FFMPEG_VERSION(56,14,100)
```

Comment:
I believe all new preprocessor checks should be shorthanded to something like `#ifdef ENABLE_FFMPEG_HW`

Reply:
Thank you for suggestion. Should I shorthand this directive to smth like ENABLE_FFMPEG_HW_SELECTION, because ffmpeg hw can be initialized implicitly with the default device?

Reply:
Yes, sounds good.